webapp/src/hooks/useAuth.tsx

```diff
@@ -92,7 +92,7 @@ export function AuthProvider({
             return;
           } 
           setUser(response);
-          if(!user.memberships.organization) {
+          if(!user?.memberships?.organization) {
```

Comment:
on peut déclarer plus haut un boolean du style

```tsx
const isOrganization = user?.memberships?.organization;
```

et utiliser comme 
```suggestion
          if(!isOrganization) {
```

c'est plus lisible, non?